docs/manual-installation.md

```diff
@@ -0,0 +1,123 @@
+# Manual installation tutorial
+
+## Step by step
+
+We're starting with an Ubuntu 16.04 LTS. This is an opinionated guy and YMMV, so if you're free to use the technology that you fell most comfortable on. If you have any doubts and you're blocked you can go and ask on [our Gitter](https://gitter.im/decidim/decidim). We recommend that you follow some Ruby on Rails tutorials (like [Getting Started with Ruby on Rails](http://guides.rubyonrails.org/getting_started.html)) and have some knowledge on how gems and engines work.
+
+On this tutorial we'll see how to install rbenv, PostgreSQL and Decidim, and how to configure everything together.
+
+### Installing rbenv
+
+First we're going to install rbenv, for managing various ruby versions. We're following the guide from [DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-install-ruby-on-rails-with-rbenv-on-ubuntu-16-04).
+
+```bash
+sudo apt-get update
+sudo apt-get install -y build-essential autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm3 libgdbm-dev
+git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+source ~/.bashrc
+git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+rbenv install 2.5.0
+rbenv global 2.5.0
+echo "gem: --no-document" > ~/.gemrc
+gem install bundler
+```
+
+### Installing Decidim
+
+Next, we need to install the `decidim` gem:
+
+```bash
+gem install decidim
+```
+
+Afterwards, we can create an application with the nice `decidim` executable, where `decidim_application` is your application name (ie DecidimBarcelona):
+
+```bash
+decidim decidim_application
+cd decidim_application
+bundle install
+```
+
+We recommend that you save it all on Git.
+
+```bash
+git init .
+git commit -m "Initial commit. Generated with Decidim 0.X https://decidim.org"
+```
+
+### Installing PostgreSQL
+
+Now we're going to install PostgreSQL for the database:
+
+```bash
+sudo apt-get install -y postgresql libpq-dev
```

Comment:
If `libpq-dev` is not installed, the previous `bundle install` will fail so probably this step should be before the generation of the application...

Reply:
You can do this with a oneliner :)

```bash
sudo -u postgres psql -c "CREATE USER decidim_app WITH SUPERUSER CREATEDB NOCREATEROLE PASSWORD 'decidim_app'"
```

Not that I used `NOCREATEROLE` because the DB user does not need role creation privileges.

Reply:
Ok, great feedback, I'll change it

Reply:
> If libpq-dev is not installed, the previous bundle install will fail so probably this step should be before the generation of the application...

I've moved the postgres installation so it's before decidim installation

Reply:
> You can do this with a oneliner :)

Changed it!